src/android/TP6mAggregate.jsx

```diff
@@ -114,14 +197,14 @@ async function pullAggregate({
     },
   );
   const result = window(
-    { daily, g5Reference },
+    { daily, refMin },
     {
       edges: ['test', 'platform', 'pushDate'],
       value: (row) => {
-        const { daily, g5Reference } = row;
+        const { daily, refMin } = row;
 
         return round(
-          selectFrom(daily, g5Reference)
+          selectFrom(daily, refMin)
```

Comment:
now we can use `refMin` instead of the original g5Reference data.  In this case we are ensuring we have both reference data and sample data.